[시저] pull request 테스트

caesar1030 opened the pull request:
draft pull request 테스트

caesar1030 committed:
학습계획 작성

caesar1030 committed:
Revert "학습계획 작성"

This reverts commit 7d5ea73dbbcf935393c39228d80a6df6990fd210.

caesar1030 committed:
수정 readme.md

crongro reviewed (approved):
해야할일이나 학습내용은, 아직 구체화 하지 않아도 됩니다.
대신 매일 업데이트하면서 좀더 구체화해보세요.

caesar1030 committed:
Feat: "header 마크업 작성"

css 작업을 위한 기본적인 마크업 작성.

caesar1030 committed:
Design: "수정 header 디자인"

기본적인 레이아웃 배치

caesar1030 committed:
Design: "수정 index.css 파일"

헤더 부분 css 작성 완료.

caesar1030 committed:
Design: "수정 index.html style.css 파일"

이미지 추가

caesar1030 committed:
Design: "수정 index.html, style.css 파일"

하얀색 아이콘을 사용하기 위해 이미지를 png 파일에서 svg 파일로 변경.

caesar1030 committed:
Fix: "수정 logo세로 정렬"

logo에 a태그를 감싼 후 세로 중앙 정렬이 되지 않는 문제 해결.

caesar1030 committed:
Feat: "로그인 팝업 기능 구현"

페이지 접속 후 1초 후에 팝업이 발생하고 로그인 창에 마우스를 올려놓을
경우,
팝업 사라지는 기능 구현

caesar1030 committed:
Feat: "로그인 레이어 팝업 기능 구현"

헤더의 로그인 부분에 hover시 레이어 팝업 기능 구현."

caesar1030 committed:
Feat: "dimmed 처리 기능 구현"

모달 혹은 레이어 팝업 시 dimmed 처리를 위한 레이어 생성.

caesar1030 committed:
Fix: "딤처리 오류 수정"

딤처리시 헤더 부분까지 레이어가 쌓이는 오류 수정

caesar1030 committed:
Feat: "배송지 레이어 팝업 기능 구현"

배송지에 hover시 레이어 팝업 및 딤처리 기능 구현

caesar1030 committed:
Comment: "누락된 주석 추가"

shipping address 레이어 팝업에 대한 누락된 주석 추가

caesar1030 committed:
Design: "nav css 수정"

기존의 경우, nav 박스 영역이 아닌 글자에 대해서만 anchor역할 (마우스
올릴 시 커서 포인터)을 하였지만 좀 더 좋은 ux를 위해 nav 하나의 박스에
anchor 역할 부여.

crongro commented on assets/fonts/web/static/pretendard-subset.css:
```diff
@@ -0,0 +1,71 @@
+http://scripts.sil.org/OFL
+*/
+
+@font-face {
```
웹폰트 성능상 단점도 알아보세요.
화면이 이뻐지겠고만요~

crongro commented on index.html:
```diff
@@ -0,0 +1,182 @@
+          </div>
+        </a>
+
+        <div class="pop-up pop-up--shipping-address hidden">
```
클래스 네이밍의 어떤 규칙이 있는것 같군요.
일관성을 잘 지키신 듯 합니다.

crongro commented on index.js:
```diff
@@ -0,0 +1,58 @@
+  }, delay);
+}
+
+function main() {
```
main 함수가 여러가지를 담고 있는데,main이다보니 자주 변경되고 더 커질 수도 있겠죠?
main 을 더 가볍게 유지하는 방법을 생각해보세요

crongro commented on styles.css:
```diff
@@ -0,0 +1,263 @@
+/* 글로벌로 뺀다? */
+
+/* 명시도 높이기 */
+.hidden.hidden {
```
아 이렇게 높인다는거 첨봐요~ ㅎ
좋은건지 나쁜건지 모르겠어요 ㅋ

crongro commented on styles.css:
```diff
@@ -0,0 +1,263 @@
+}
+
+.nav__item {
+  list-style: none;
```
요런건  li {} 안에다가 reset 하듯이 설정하는것도 좋을듯.

crongro reviewed (approved):
웹폰트도 시도해보시고, js코드가 보여서 반갑군요. 

리뷰참고해서 이어서 개발해보세요.
승인&머지합니다.
